packages/data-explorer-ui/src/components/common/Button/button.styles.ts

```diff
@@ -59,3 +59,13 @@ export const TextButtonPrimary = styled(MButtonBase)`
     text-decoration: underline;
   }
 `;
+
+// White text button
+export const TextButtonWhite = styled(MButtonBase)`
+  color: ${({ theme }) => theme.palette.common.white};
+  font-size: 13px;
+  font-weight: ${({ theme }) => theme.typography[textBody500].fontWeight};
+  line-height: ${({ theme }) => theme.typography[textBody500].lineHeight};
+  line-height: 1;
+  text-decoration: underline;
```

Comment:
Add the style `vertical-align: unset` - to override the unwanted alignment of the text from MuiButtonBase.